app/views/course/forum/forums/_controls.html.slim

```diff
@@ -3,12 +3,13 @@ span.btn-group
     = link_to unsubscribe_course_forum_path(current_course, @forum),
               title: t('course.forum.forums.unsubscribe.tag'), class: ['btn', 'btn-danger'],
               method: :delete do
-      = fa_icon 'eye-slash'.freeze
+      = fa_icon 'heart-o'.freeze
```

Comment:
I just change the picture of the forum subscribe button in this PR. And I'll modify these buttons in next PR following the method you provided. http://stackoverflow.com/questions/20821171/bootstrap-button-group-styling-with-button-to
@lowjoel